services/prometheus/scrape_configs.go

```diff
@@ -25,40 +25,34 @@ import (
 	"strings"
 	"time"
 
-	"github.com/pkg/errors"
-	"github.com/prometheus/common/model"
+	config "github.com/percona/promconfig"
 
 	"github.com/percona/pmm-managed/models"
-	config_util "github.com/percona/pmm-managed/services/prometheus/internal/common/config"
-	"github.com/percona/pmm-managed/services/prometheus/internal/prometheus/config"
-	sd_config "github.com/percona/pmm-managed/services/prometheus/internal/prometheus/discovery/config"
-	"github.com/percona/pmm-managed/services/prometheus/internal/prometheus/discovery/targetgroup"
 )
 
-const addressLabel = model.LabelName(model.AddressLabel)
-
 // scrapeTimeout returns default scrape timeout for given scrape interval.
-func scrapeTimeout(interval time.Duration) model.Duration {
+// nolint:gomnd
```

Comment:
> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.

From https://github.com/golangci/golangci-lint#nolint

Reply:
Fixed